cmflib/commands/artifact/list.py

```diff
@@ -17,69 +17,217 @@
 import argparse
 import os
 import pandas as pd
+import textwrap
 
+from tabulate import tabulate
 from cmflib.cli.command import CmdBase
 from cmflib import cmfquery
+from cmflib.dvc_wrapper import dvc_get_config
 
 class CmdArtifactsList(CmdBase):
-    def update_dataframe(self, df):
-        for c in df.columns:
-            if c.startswith('custom_properties_'):
-                df.rename(columns = {c:c.replace('custom_properties_','')}, inplace = True)
-            else:
-                df = df.drop(c, axis = 1)
-        return df
+    def update_dataframe(self, df: pd.DataFrame, is_long: bool) -> pd.DataFrame:
+        """
+        Updates the dataframe to fit the specified table format based on the length option.
+
+        Parameters:
+        - df: DataFrame to be updated.
+        - is_long: Boolean indicating whether to use long option or not.
+
+        Returns:
+        - Updated DataFrame with selected columns.
+        """
+        if is_long:
+            updated_columns = ["id", "name"] + [ col for col in df.columns if not (col == "id" or col == "name")]
+        else:
+            updated_columns = ["id", "name"] + [ col for col in df.columns if col.startswith('custom_properties_')]
+            # Limit to a maximum of 5 columns.
+            if len(updated_columns) > 5:
+                updated_columns = updated_columns[:5]
```

Comment:
I want to confirm if this slicing of '5' will ever change in future or if this is absolute ?

Reply:
As mentioned above, for now we are keeping constant as discussed in meeting.